core/executor/src/precompiles/ckb_mbt_verify.rs

```diff
@@ -0,0 +1,86 @@
+use ckb_types::packed;
+use ckb_types::prelude::Pack;
+use ckb_types::utilities::{merkle_root, MerkleProof};
+use ethers::abi::AbiDecode;
+use ethers::contract::{EthAbiCodec, EthAbiType};
+use evm::executor::stack::{PrecompileFailure, PrecompileOutput};
+use evm::{Context, ExitError, ExitSucceed};
+
+use protocol::types::H160;
+
+use crate::err;
+use crate::precompiles::{axon_precompile_address, PrecompileContract};
+
+#[derive(EthAbiCodec, EthAbiType, Clone, Debug, PartialEq, Eq)]
+pub struct VerifyProofPayload {
+    pub transactions_root:     [u8; 32],
+    pub witnesses_root:        [u8; 32],
+    pub raw_transactions_root: [u8; 32],
+    pub indices:               Vec<u32>,
+    pub lemmas:                Vec<[u8; 32]>,
+    pub leaves:                Vec<[u8; 32]>,
+}
```

Comment:
how can I get this payload via CKB-RPC?

Reply:
Use [https://github.com/nervosnetwork/ckb/blob/develop/rpc/README.md#method-get_transaction_proof](https://github.com/nervosnetwork/ckb/blob/develop/rpc/README.md#method-get_transaction_proof) RPC

Reply:
how can I verify a cell belongs to a transaction? and to verify the transaction itself, I can verify its hash is included in the `transacitons_root`?